packages/framer-motion/src/render/svg/SVGVisualElement.ts

```diff
@@ -67,4 +70,9 @@ export class SVGVisualElement extends DOMVisualElement<
     ): void {
         renderSVG(instance, renderState, styleProp, projection)
     }
+
+    mount(instance: SVGElement) {
+        this.isSVGTag = instance.tagName.toLowerCase() === "svg"
```

Comment:
I'm on the fence about this suggestion but should we do:

`isSVGTag(tag: string): boolean`? We do use it in 3 places 🤔